tests/test_KafkaError.py

```diff
@@ -31,6 +33,22 @@ def test_error_cb():
     assert seen_all_brokers_down
 
 
+def test_fatal():
+    """ Test fatal exceptions """
+
+    # Configure an invalid broker and make sure the ALL_BROKERS_DOWN
+    # error is seen in the error callback.
+    p = Producer({'error_cb': error_cb})
+
+    with pytest.raises(KafkaException) as exc:
+        KafkaError._test_raise_fatal()
+    err = exc.value.args[0]
```

Comment:
is `exc` in scope here? doesn't seem right

Reply:
Yeah, I had the same reaction initially, but this is according to docs:
https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions